src/Command/Apps/GitIntegration/CreateGitIntegrationCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Pagely\AtomicClient\Command\Apps\GitIntegration;
+
+use Pagely\Client\AuthApi;
+use Pagely\Client\Command\Command;
+use Pagely\Client\Command\OauthCommandTrait;
+use Pagely\Client\AppsApi\AppsClient;
```

Comment:
I believe you need to fix the namespaces.  We don't include mgmt-client in atomic-client.
Also, the classes may not be in the same place.